manager2/src/app/admin/projects/projects.component.ts

```diff
@@ -310,7 +310,7 @@ export class ProjectsComponent implements OnInit {
 
     reject_project(project) {
         this.reset_msgs()
-        this.projectService.delete_pending(project.id).subscribe(
+        this.projectService.delete_pending(project.uuid).subscribe(
```

Comment:
If the admin rejects a pending project, it is now done using the uuid, in case several projects with the same name are asked by the users.